package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
-  "version": "2.5.8",
+  "version": "2.5.9",
```

Comment:
Because it introduces breaking changes (renaming the component), this is actually a major version bump.

Reply:
Updated, thanks!